README.md

```diff
@@ -1,2 +1,6 @@
 # ML-python-pytorch-lstm-gpu-service
-Jenkins X quick start for training and deploying LSTM Neural Network Classification using the PyTorch library
+
+Quickstart project for executing an IMDB classifier using the PyTorch framework on a GPU.
+
+Running `pip install requirements.txt` and then python `app.py` will start the app on localhost where the user can send POST requests to perform inference.
+
```

Comment:
Can you include a bit of info and example of what format / type of POST request needs to be sent (ie json with text ; what the endpoint is ...)